connect-examples/v2/csharp_payment/Pages/index.cshtml

```diff
@@ -11,55 +11,66 @@
         window.locationId = "@Model.LocationId";
     </script>
     <!-- link to the local SqPaymentForm initialization -->
-    <script type="text/javascript" src="~/js/sqpaymentform.js"></script>
+    <script type="text/javascript" src="https://cdn.rawgit.com/square/connect-api-examples/master/templates/web-ui/payment-form/custom/sq-payment-form.js"></script>

```

Comment:
Do you want to put the relative path to the form here? I am guessing this link will always pull the latest `sq-payment-form.js` from master branch which can lead to versions out of sync for the code(SHA1) vs payment form(SHA2). My concern is if the template changes. Could it potentially break the example?

Reply:
Is it possible to since the files are hosted above where the applications themselves are saved? I tried for awhile on the Rails example and couldn't get it to work (and people on #ruby-newbies told me I should either 1) publicly host or 2) move it to inside the rails directory). So I opted for this way, but if it's possible to do this, then yes I agree it would be better.

Reply:
okay that makes sense. We probably won't make major changes to the payment form often. I agree with this approach.